src/boards/rainbow_esp.h

```diff
@@ -19,6 +19,16 @@
 static uint8 const NO_WORKING_FILE = 0xff;
 static uint8 const NUM_FILE_PATHS = 3;
 static uint8 const NUM_FILES = 64;
+static uint8 const NUM_NETWORKS = 3;
+static uint8 const NUM_FAKE_NETWORKS = 5;
+static uint8 const SSID_MAX_LENGTH = 32;
+static uint8 const PASS_MAX_LENGTH = 64;
+
+typedef struct
+{
+	std::string ssid;
+	std::string pass;
+} networkInfo;
```

Comment:
Coding style:
```
struct NetworkInfo {
    std::string ssid;
    std::string pass;
};
```
Note that the type name is to be changed (initial is capitalized)